percona/controller/pgbackup/controller.go

```diff
@@ -169,22 +192,43 @@ func getDestination(pg *v2.PerconaPGCluster, pb *v2.PerconaPGBackup) string {
 }
 
 func getBackupType(job *batchv1.Job) v2.PGBackupType {
-	v := job.GetLabels()[naming.LabelPGBackRestCronJob]
-	if v == "" {
-		v = job.GetLabels()[naming.LabelPGBackRestBackup]
+	var backupContainer corev1.Container
+	for _, container := range job.Spec.Template.Spec.Containers {
+		if len(container.Command) > 0 && container.Command[0] == "/opt/crunchy/bin/pgbackrest" {
```

Comment:
can't we use container name to check this?

Reply:
https://github.com/percona/percona-postgresql-operator/pull/524/commits/8f575870c5e566a65e8008634c4f898287498369